doc/getting-started/apl-nuc.rst

```diff
@@ -21,10 +21,14 @@ described in :ref:`hardware`, are currently supported for ACRN development:
 Connecting to the serial port
 =============================
 
-If you don't need a serial console you can ignore this section. If you're
-using a KBL NUC and you need a serial console, you'll need to prepare
-an RS232 cable to connect to the KBL NUC serial port header, as shown
-below:
+If you don't need a serial console you can ignore this section.
```

Comment:
This section only applies to the KBL NUC, right?  So let's rewrite this as:
```
Connecting to the KBL NUC serial port
=====================================

If you don't require a serial console, you can ignore this section.

Neither the APL or KBL NUCs present an external serial port interface.
However, the KBL NUC does have a serial port header you 
can expose with a serial DB9 header cable.  You can build this cable
yourself, referring to the `KBL NUC product specification
<https://www.intel.com/content/dam/support/us/en/documents/boardsandkits/NUC7i5DN_TechProdSpec.pdf>`__
as shown below:

.. figure:: images/KBL-serial-port-header.png
   :align: center

.. figure:: images/KBL-serial-port-header-to-RS232-cable.jpg
   :align: center

Or you can `purchase
<https://www.amazon.com/dp/B07BV1W6N8/ref=cm_sw_r_cp_ep_dp_wYm0BbABD5AK6>`_
such a cable.

You'll also need an `RS232 DB9 female to USB cable
<https://www.amazon.com/Adapter-Chipset-CableCreation-Converter-Register/dp/B0769DVQM1>`__,
or an `RS232 DB9 female/female (NULL modem) cross-over cable
<https://www.amazon.com/SF-Cable-Null-Modem-RS232/dp/B006W0I3BA>`__
to connect to your host system.

```
Can you verify the Amazon cable links I've provided are correct?  (The one @wuxyintel provided
was to a Chinese site.)

Are we concerned we're telling folks to disassemble their NUC to add this header cable? Is it possible to close the NUC case with this header cable connected? 

The cable picture you have show all 9 wires connected, but the header spec picture only shows you need 3 wires connected?  If you build the cable yourself, and you only need three wires, you should show that in the cable picture.  Otherwise, you should show how the rest of the wires should be connected between the header and DB9 connector.

Reply:
> Can you verify the Amazon cable links I've provided are correct? (The one @wuxyintel provided
> was to a Chinese site.)

This looks like the right cable. We could also add this link if we want to make sure: https://www.gorite.com/serial-db9-header-cable-for-nuc-dawson-canyon

> Are we concerned we're telling folks to disassemble their NUC to add this header cable? Is it possible to close the NUC case with this header cable connected?

No, that cable is meant to fit in the enclosure and there is a metal plate at the back that one can easily cut out (it's pre-cut) to attach the DB9 connector at the back (take a look at the other picture and you'll see it's in fact a very clean set-up).

Reply:
nice!  (I don't have a KBL NUC to check).